src/ui/screens/onboarding/OnboardingDataSlide.qml

```diff
@@ -0,0 +1,105 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+import QtQuick 2.15
+import QtQuick.Layouts 1.15
+
+import Mozilla.Shared 1.0
+import components 0.1
+
+ColumnLayout {
+    id: root
+
+    signal nextClicked()
+
+    spacing: 0
+
+    MZHeadline {
+        Layout.topMargin: MZTheme.theme.vSpacing
+        Layout.leftMargin: MZTheme.theme.windowMargin * 2
+        Layout.rightMargin: MZTheme.theme.windowMargin * 2
+        Layout.fillWidth: true
+
+        text: MZI18n.OnboardingDataSlideHeader
+        horizontalAlignment: Text.AlignLeft
+    }
+
+    MZInterLabel {
+        Layout.topMargin: 8
```

Comment:
Are absolute margin values okay or should these be relative? I see a few different values throughout this implementation.

Reply:
if by "relative", you mean "should we be using references to values in `theme.js`", then you can find some more info on this in a [discussion between Vinoo and I above](https://github.com/mozilla-mobile/mozilla-vpn-client/pull/7930#discussion_r1320401173) but in essence, I think literal values are fine if they aren't coupled/associated with anything else. This is just spacing between components on a page and isn't really re-usable in any way. 

If you meant something else, let me know!